frontend/server/src/DAO/Contests.php

```diff
@@ -132,7 +136,7 @@ final public static function getByTitle(string $title) {
     final public static function getByAliasWithExtraInformation(string $alias): ?array {
         $sql = '
                 SELECT
-                    c.*,
+                    join(', ', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES)),
                     p.scoreboard_url,
```

Comment:
Y también acá (la concatenación se quedó en L139)
```suggestion
                    'p.scoreboard_url,
```